autopush/websocket.py

```diff
@@ -896,6 +904,12 @@ def error_notification_overload(self, fail):
         d = self.deferToLater(randrange(5, 60), self.process_notifications)
         d.addErrback(self.trap_cancel)
 
+    def error_message_overload(self, fail):
+        """errBack for handling excessive messages per UAID"""
+        fail.trap(MessageOverloadException)
+        self.ap_settings.router.drop_user(self.ps.uaid)
```

Comment:
Ah, looks like it should be in a force_retry to make sure it runs even if a client drops.